Formula/n/newnode.rb

```diff
@@ -0,0 +1,69 @@
+class Newnode < Formula
+  desc "Web proxy that uses a distributed p2p network to circumvent censorship"
+  homepage "https://www.newnode.com/newnode-vpn"
+  url "https://github.com/clostra/newnode.git",
+    tag: "2.1.5", revision: "18a3f713267b2a08e34cbe04253df891889997a2"
+  license "GPL-2.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "coreutils" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  on_macos do
+    depends_on xcode: ["9.3", :build]
+  end
+
+  on_linux do
+    depends_on "gnu-sed" => :build
+    depends_on "grep" => :build
+    depends_on "llvm" => :build
+    depends_on "wget"
+  end
+
+  def install
+    if OS.linux?
+      inreplace "https_wget.c", "/usr/bin/wget", "#{Formula["wget"].opt_bin}/wget"
+      ENV.prepend_path "PATH", Formula["gnu-sed"].opt_libexec/"gnubin"
+      ENV["HOMEBREW_CC"] = "#{Formula["llvm"].opt_bin}/clang"
+      ENV["HOMEBREW_CXX"] = "#{Formula["llvm"].opt_bin}/clang++"
+    end
+    system "./build.sh"
+    bin.install "client" => "newnode"
+    path = (var/"cache/newnode")
+    path.mkpath
+    path.chmod 0775
```

Comment:
This means that every install/update will make a new cache. Is that desirable?

Reply:
can you point us the installation doc reference on this?

Reply:
No, currently it will make a new cache folder as part of the install and overwrite the old one.

Reply:
I believe that's even better.

Reply:
And if people are using the tool when running `brew upgrade` it'll keep working?

Reply:
It should, if peers file is invalid it just won't be loaded. There's no parsing, just raw ip addresses.

Reply:
Either way works, saving if not saving. We will handle the update smoothly if that interface ever changes. The code is intentionally simple and shouldn't fail.